pkg/controller/migration/convert/network_test.go

```diff
@@ -755,6 +755,149 @@ var _ = Describe("Convert network tests", func() {
 		})
 
 		Context("Calico CNI config flags", func() {
+			Describe("migrate tuning setting", func() {
+				It("sysctl tuning in config", func() {
```

Comment:
I think this test is written correct but because of the inverted validation logic it fails.